tools/python/boutiques/bosh.py

```diff
@@ -277,6 +281,37 @@ def evaluate(*params):
     return query_results[0] if len(query_results) == 1 else query_results
 
 
+def test(*params):
+
+    parser = ArgumentParser("Perform all the tests defined within the given descriptor")
+    parser.add_argument("descriptor", action="store", help="The Boutiques descriptor.")
+    result = parser.parse_args(params)
+
+    # Generation of the invocation schema (and descriptor validation).
+    invocation(result.descriptor)
+    
+    # Extraction of all the invocations defined for the test-cases.
+    descriptor = json.loads(open(result.descriptor).read())
+    
+    if (not descriptor.get("tests")):
+        raise SystemExit("No test found in descriptor")
```

Comment:
Yes that's a good point.

I'll make the modifications so that bosh test simply return 0 in this case.
0 is the exit code returned by pytest if all the tests have passed successfully.